AntennaTracker/Parameters.cpp

```diff
@@ -448,6 +456,14 @@ const AP_Param::Info Tracker::var_info[] = {
     // @Increment: 0.5
     // @User: Advanced
 
+    // @Param: YAW2SRV_PDMX
+    // @DisplayName: PD Sum Maximum
```

Comment:
I think this Display name should start with the axis.  So perhaps "Yaw PD Sum Maximum".  Same for other new parameter descriptions too.  Txs